docs/linting/typed-linting/MONOREPOS.md

```diff
@@ -40,6 +40,7 @@ Be sure to update your `.eslintrc.js` to point at this new config file.
 
 The `parserOptions.project` option introduced in [Linting with Type Information](../TYPED_LINTING.md) accepts an array of relative paths.
 Paths may be provided as [Node globs](https://github.com/isaacs/node-glob/blob/f5a57d3d6e19b324522a3fa5bdd5075fd1aa79d1/README.md#glob-primer).
+And the path resolution is performed in order from the beginning of the array.
```

Comment:
Hmm, this isn't quite saying what I think we want it to. What happens is that the _first matching project_ is what gets used.

1. Normalizing project input paths: https://github.com/typescript-eslint/typescript-eslint/blob/09c04fb0d5a5dee0dc266abb2e21aa9a1c489712/packages/typescript-estree/src/parser.ts#L308-L311
2. Non-single-run usage: https://github.com/typescript-eslint/typescript-eslint/blob/09c04fb0d5a5dee0dc266abb2e21aa9a1c489712/packages/typescript-estree/src/create-program/useProvidedPrograms.ts#L22-L28

How does this phrasing look to you?

```suggestion
For each file being linted, the first matching project path will be used as its backing TSConfig.
```

Reply:
@JoshuaKGoldberg 
Thank you for explaining in detail!
Now I understand that the phrasing you suggested is better!